### ISSUE TITLE
New UDF: Label(User Defined Function Body) is not associated with JSON edit field present under 'New UDF1' Tab section of the page.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1050]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1242591

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure.
Navigate to "New UDF" menu item from the drop down select it.
Navigate through all the controls under "New UDF 1".
Navigate to 'User Defined Function Body' JSON edit field.
Press F12 key to open Developers tool and inspect the code.
Verify the issue.

Actual Result:
Label(User Defined Function Body) is not associated with JSON edit field present under 'New UDF1' Tab section of the page due to which screen reader is not reading label information.

Expected Result:
Label(User Defined Function Body) should be associated with JSON edit field present under 'New UDF1' Tab section of the page.

Note:
This issue also exists under 'New Trigger' tab section of the page for similar control.
Refer to the attachment: [MAS3.3.2-Label not associated.]

**Comment by ato9000** [comment]
![image](https://user-images.githubusercontent.com/8519407/132707683-514fd786-7daf-46c2-add5-c906597d9387.png)

![image](https://user-images.githubusercontent.com/8519407/132707733-1e978487-af4a-4b09-97e7-9ce80811f262.png)
